app/livechat/server/api/v1/message.js

```diff
@@ -34,6 +34,10 @@ API.v1.addRoute('livechat/message', {
 				throw new Meteor.Error('invalid-room');
 			}
 
+			if (!room.open) {
+				throw new Meteor.Error('chat-closed');
```

Comment:
Please, replace the `chat-closed` to `room-closed`, according we do here:
https://github.com/RocketChat/Rocket.Chat/blob/develop/app/livechat/server/api/v1/room.js#L56

Reply:
Done 😄